config/routes.rb

```diff
@@ -223,6 +223,7 @@
 
   # Help
   get 'help' => 'help#faq', :as => 'help'
+  get 'help/:topic' => 'help#faq'
```

Comment:
@svbergerem, that's frustrating. https://github.com/thoughtbot/hound/pull/617 should help with the disabling of languages. The jshint config change should work, please let me know if it doesn't.